modules/api/src/test/scala/vinyldns/api/domain/dns/DnsConversionsSpec.scala

```
@@ -336,6 +345,11 @@ class DnsConversionsSpec
       val result = rightValue(toDnsRRset(testTXT, testZoneName))
       verifyMatch(result, testTXT)
     }
+
+    "fail to convert a bad SPF record set" in {
```

SPF might need to be updated like TXT record data, but I'm putting this test in for now to confirm that we aren't swallowing errors that might occur during the `toDnsRRset` function. we can create a separate issue and PR to fix SPF and then this test can be removed or modified.

---

SPF is not implemented correctly, I agree that is a separate PR

---

oh yeah, SPF isnt even supposed to be a type, its a sub of TXT...

---

for later